docs/support_files/world.md

```diff
@@ -4,20 +4,20 @@
 The default world constructor is:
 
 ```javascript
-function World({attach, parameters}) {
-  this.attach = attach
-  this.parameters = parameters
+function World({ attach, parameters }) {
+  this.attach = attach;
```

Comment:
Please remove the semicolons. We don't use them in the codebase, so I'd prefer to be consistent in code examples.

Reply:
Done